src/validation.h

```diff
@@ -860,6 +859,10 @@ class ChainstateManager
 
     const CChainParams& GetParams() const { return m_options.chainparams; }
     const Consensus::Params& GetConsensus() const { return m_options.chainparams.GetConsensus(); }
+    bool IsCheckBlockIndex() const
```

Comment:
In commit "Move ::fCheckBlockIndex into ChainstateManager" (fa41ab525b98df20bf84946cfcc9289dfb21113d)

"IsCheck" sounds like bad grammar, maybe "ShouldCheck" or "DoCheck" or just "Check" would be better.

Reply:
Thanks, renamed